### PR TITLE
Update entrypoint script to work with pachyderm helm values

### PR DESCRIPTION
**Pull request by smalyala**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -3,156 +3,74 @@
...
+    listen_port = ${LISTEN_PORT:-5432}
+    auth_file = ${AUTH_FILE:-$PG_CONFIG_DIR/userlist.txt}
+    auth_type = ${AUTH_TYPE:-md5}
+    ${POOL_MODE:+pool_mode = ${POOL_MODE}\n}\
+    ${MAX_CLIENT_CONN:+max_client_conn = ${MAX_CLIENT_CONN}\n}\
+    ignore_startup_parameters = ${IGNORE_STARTUP_PARAMETERS:-extra_float_digits}
+    admin_users = ${ADMIN_USERS:-postgres}
+    ${IDLE_TRANSACTION_TIMEOUT:+idle_transaction_timeout = ${IDLE_TRANSACTION_TIMEOUT}\n}\
+
+    # TLS settings
```
Comment (tybritten) [inline entrypoint.sh:67]: ah you need to add the tls files from the tls checks here

File: entrypoint.sh [hunk 2]
```diff
@@ -3,156 +3,74 @@
...
+    ${CLIENT_TLS_SSLMODE:+client_tls_sslmode = ${CLIENT_TLS_SSLMODE}\n}\
+    ${SERVER_TLS_SSLMODE:+server_tls_sslmode = ${SERVER_TLS_SSLMODE}\n}\
+    ################## end file ##################
```
Comment (tybritten) [inline entrypoint.sh:72]:
```suggestion
${PGBOUNCER_SERVER_TLS_CA_FILE:+server_tls_ca_file = ${PGBOUNCER_SERVER_TLS_CA_FILE}\n}\
${PGBOUNCER_SERVER_TLS_KEY_FILE:+server_tls_key_file = ${PGBOUNCER_SERVER_TLS_KEY_FILE}\n}\
${PGBOUNCER_SERVER_TLS_CERT_FILE:+server_tls_cert_file = ${PGBOUNCER_SERVER_TLS_CERT_FILE}\n}\
    ################## end file ##################
```

**Review by tybritten — CHANGES_REQUESTED** [review]

**Comment by tybritten** [comment]
i think maybe we should add an 'if' to do if any of those server tls vars are set, to set the SERVER_TLS_SSLMODE

File: entrypoint.sh [hunk 3]
```diff
@@ -3,156 +3,77 @@
...
+    auth_file = ${AUTH_FILE:-$PG_CONFIG_DIR/userlist.txt}
+    auth_type = ${AUTH_TYPE:-md5}
+    ${POOL_MODE:+pool_mode = ${POOL_MODE}\n}\
+    ${MAX_CLIENT_CONN:+max_client_conn = ${MAX_CLIENT_CONN}\n}\
+    ignore_startup_parameters = ${IGNORE_STARTUP_PARAMETERS:-extra_float_digits}
+    admin_users = ${ADMIN_USERS:-postgres}
+    ${IDLE_TRANSACTION_TIMEOUT:+idle_transaction_timeout = ${IDLE_TRANSACTION_TIMEOUT}\n}\
+
+    # TLS settings
+    ${CLIENT_TLS_SSLMODE:+client_tls_sslmode = ${CLIENT_TLS_SSLMODE}\n}\
```
Comment (tybritten) [inline entrypoint.sh]:
```suggestion
```

Don't need client

**Comment by smalyala** [comment]
So something like this: `# TODO: Include SERVER_TLS_SSLMODE in the pgbouncer config only if any of the server TLS file env vars are set`

**Review by tybritten — APPROVED** [review]

**Pull request merged**